wadas/ui/configure_actuators_dialog.py

```diff
@@ -363,21 +363,28 @@ def start_actuator_server(self):
             )
         self._setup_logger()
         self.ui.pushButton_stop_server.setEnabled(True)
+        self.ui.pushButton_start_server.setEnabled(False)
+        self.ui.buttonBox.setEnabled(False)
```

Comment:
Cancel button shall be always enabled. Please add signal to handle clicked() logic when FTP is running so that you can trigger server shutdown